docs/codeql/codeql-language-guides/basic-query-for-cpp-code.rst

```diff
@@ -14,62 +16,65 @@ The query we're going to run performs a basic search of the code for ``if`` stat
 
    if (error) { }
 
-Running the query
------------------
-
-#. In the main search box on LGTM.com, search for the project you want to query. For tips, see `Searching <https://lgtm.com/help/lgtm/searching>`__.
+Finding a CodeQL database to experiment with
+--------------------------------------------
 
-#. Click the project in the search results.
+Before you start writing queries, you need a CodeQL database to run them against. The simplest way to do this is to download a database directly from GitHub.com.
 
-#. Click **Query this project**.
+#. In Visual Studio Code, click the **QL** icon |codeql-ext-icon| in the left sidebar to display the CodeQL extension. 
 
-   This opens the query console. (For information about using this, see `Using the query console <https://lgtm.com/help/lgtm/using-query-console>`__.)
+#. Click **From GitHub** or the GitHub logo |github-db| at the top of the CodeQL extension to open an entry field.
 
-   .. pull-quote::
+#. Copy the URL for the repository into the field and press the keyboard **Enter** key.
```

Comment:
❓ Do we want to suggest an example repository here? For example, https://github.com/protocolbuffers/protobuf.

Reply:
Yes, I think that's quite helpful 😄